src/components/SetKeystorePasswordDialog.vue

```diff
@@ -106,11 +107,24 @@ export default {
     passwordConfirm: "",
     showPassword: false,
     showPasswordConfirm: false,
-    passwordRule: (val) => !!val || "Password is required",
+    // passwordRule: (val) => !!val || "Password is required",
     passwordConfirmRule: (password) => (val) =>
       (!!password && password == val) || "Passwords must match.",
     recaptchaVerified: false,
     accountName: "",
+    rules: {
```

Comment:
Please move the rules to `computed`, you can refer to this: https://github.com/debionetwork/debio-frontend/blob/2a75cbf2f32063a66b52ab4dba013755935f6a49/src/views/Dashboard/Doctor/Account/index.vue#L187-L193

Reply:
done, all rules move to computed